### DISCUSSION TITLE
Improve discrimination for small net

### DISCUSSION THREAD

**Discussion by MaxLin86** [STVIR/pysot#172] [General]
I have trained a small net, and the discrimination is significantly bad than big net structure.

Even increase the "NEG_NUM" to 50（POS_NUM also 50）in my config，or increase "CLS_WEIGHT" are not working.

The idea in DaSiamRPN looks useful. How can I use "semantic negative pairs" to train? Does it provided setting in config? 

Besides, the size of search region in DaSiamRPN is iteratively growing with a constant step when failed tracking is indicated. May I ask which constant step it is？Or How do you express it in terms of a formula？

Thank you for help.

**Comment by ZhiyuanChen** [comment]
I think this could actually be a bug, it might because train.py didn't merge the NEG_NUM nor CLS_WEIGHT.
